RELEASE_NOTES

```diff
@@ -3,6 +3,47 @@ For more detailed information, please see the git log.
 
 These release notes can also be consulted at http://easybuild.readthedocs.org/en/latest/Release_notes.html.
 
+v2.1.0 (April 28th 2015)
+------------------------
+
+feature + bugfix release
+- bump required vsc-base version to 2.2.0
+- add support for only (re)generating module files: --module-only (#1018)
+    - module naming scheme API is enhanced to include det_install_subdir method
+    - FIXME docs see
```

Comment:
I still need to update the docs to cover `--module-only`, I'll update this with the correct link when I've tackled that